x-pack/build.gradle

```diff
@@ -64,4 +65,13 @@ subprojects {
         ext.projectSubstitutions["org.elasticsearch.plugin:x-pack:${snapshot}"] = snapshotProject
     }
   }
+  
+  task copyEclipseSettingsXPack(type: Copy) {
```

Comment:
We already have a method to set settings, and it "works" here only because of the order of task execution (which could change). I would rather see this done in the existing eclipse settings task in the root build.gradle, and choose which (part) of the settings to copy. In particular, it would be good to not duplicate the other parts of the settings, which are already in `buildSrc/src/main/resources/eclipse.settings`. I think we should have the two licenses there, copy the files and append the appropriate code template line in a `doLast` on the copy?